### ISSUE TITLE
ktest: allow multiple I/O pairs for the same programs

### ISSUE THREAD

**Issue by grosu** [kframework/k-legacy#23]
We want to allow the same program to have multiple tests, as we used to have with ant test in the past (or maybe it still works?). Something like this:

sum.imp
sum.imp.in.1
sum.imp.out.1
sum.imp.in.2
sum.imp.out.2
...

**Comment by andreiarusoaie** [comment]
Here I need some update from @osa1 : does this work now? If yes, please feel free to close the issue.

**Comment by osa1** [comment]
No, I'll start working on this as soon as we merge new ktest, which may happen any minute now.
